demos/python_demos/object_detection_demo/object_detection_demo.py

```diff
@@ -258,6 +254,10 @@ def main():
             presenter.drawGraphs(frame)
             frame = draw_detections(frame, objects, palette, model.labels, args.prob_threshold)
             metrics.update(start_time, frame)
+
+            if video_writer.isOpened():
+                video_writer.write(frame)
```

Comment:
There is the second part of pipeline below: `# Process completed requests` which also should save frames. The same is for segmentation.

Reply:
done.